constants/src/lib.rs

```diff
@@ -62,6 +64,12 @@ pub type CurvePoint = GenericCurvePoint<SystemCurveGroup>;
 #[cfg(feature = "mpc-types")]
 pub type AuthenticatedScalar = AuthenticatedScalarResult<SystemCurveGroup>;
 
+/// The (unbatched) proof type that the proof system generates
+pub type SystemProof = Proof<SystemCurve>;
```

Comment:
This probably belongs better elsewhere, I think it's okay to put it in the Arbitrum client for now. My reasoning: this does not parameterize the system in any way, it is _parameterized by_ other system parameters. `constants` should be kept really lean and have few dependencies -- essentially only to specify system parameters.

Reply:
sounds good, will move it into `circuit-types` along with the commitment type below if that works - just feels like a more appropriate home than the Arbitrum client